gensim/models/wrappers/fasttext.py

```diff
@@ -272,11 +282,13 @@ def load_model_params(self, file_handle):
         self.sample = t
 
     def load_dict(self, file_handle, encoding='utf8'):
-        (vocab_size, nwords, _) = self.struct_unpack(file_handle, '@3i')
+        vocab_size, nwords, _ = self.struct_unpack(file_handle, '@3i')
         # Vocab stored by [Dictionary::save](https://github.com/facebookresearch/fastText/blob/master/src/dictionary.cc)
         assert len(self.wv.vocab) == nwords, 'mismatch between vocab sizes'
         assert len(self.wv.vocab) == vocab_size, 'mismatch between vocab sizes'
-        ntokens, = self.struct_unpack(file_handle, '@q')
+        ntokens = self.struct_unpack(file_handle, '@1q')
```

Comment:
Could change this to simply `self.struct_unpack(file_handle, '@1q')  # number of tokens - unused` to remove the unused variable.